### DISCUSSION TITLE
Recently when VSK fresh installation done using release v5.0.3-nvsk branch, generateToken gives 502 error

### DISCUSSION THREAD

**Discussion by archanamakude** [Sunbird-cQube/community#860] [Q&A]
There's a 502 error while generating JWT token with cQube v5.0.3-nvsk version.

{
    "message": "An invalid response was received from the upstream server",
    "request_id": "aec82b79886f1317926fd83a2ba2df30"
}
![ingest_ms_issue](https://github.com/Sunbird-cQube/community/assets/30899552/85123aa1-ba39-40a7-92d7-0b7ecd11033c)

**Comment by bommalla** [comment]
@archanamakude 

Did you get any error during the installation ? if yes please send screenshot of the error ? if not please check the below mentioned path whether package-lock.json files is there or not.

path: cqube-devops/microservices/ingestion-ms/
cat Dockerfile

**Reply by archanamakude** [reply]
When we did the installation package-lock.json was not there in Dockerfile. But after installation we updated it in cqube-devops/microservices/ingestion-ms/ and then rebuilt the ingest_app docker container

**Reply by bommalla** [reply]
@archanamakude 

Please update status of the issue ?

**Comment by bommalla** [comment]
@archanamakude 

Even we are also tried to generate token with the same URL but facing same issue.I think some routing issue is there in your end please check.

**Comment by archanamakude** [comment]
The issue is resolved after copying root/.oci content to ingest_app container after rebuild.
Thank you for your help. @bommalla

**Reply by bommalla** [reply]
Thanks for the update @archanamakude
